src/parser.rs

```diff
@@ -786,13 +786,24 @@ impl<'tokens, 'src> Parser<'tokens, 'src> {
         value: Setting::WindowsShell(self.parse_shell()?),
         name,
       })
+    } else if name.lexeme() == Keyword::Tempdir.lexeme() {
+      Ok(Set {
+        value: Setting::Tempdir(self.parse_path()?),
```

Comment:
I think this can just be inlined, also the value of `Seting::Tempdir` can be a `String`:

```suggestion
        value: Setting::Tempdir(self.parse_string_literal()?.cooked),
```